beginner_source/quickstart/build_model_tutorial.py

```diff
@@ -0,0 +1,193 @@
+#
+# Example:
+#
+
+device = 'cuda' if torch.cuda.is_available() else 'cpu'
+print('Using {} device'.format(device))
+
+##############################################
+# __init__
+# -------------------------
+#
+# The ``init`` function inherits from ``nn.Module`` which is the base class for 
+# building neural network modules. This function defines the layers in your neural network
+# then it initializes the modules to be called in the ``forward`` function.
+# 
+
+##############################################
+# The Model Module Layers
+# -------------------------
+#
+# Lets break down each model layer in the FashionMNIST model.
+#
+
+##################################################
+# `nn.Flatten <https://pytorch.org/docs/stable/generated/torch.nn.Flatten.html>`_ 
+# -----------------------------------------------
+#
+# First we call nn.Flatten to reduce tensor dimensions to one.
+#
+# From the docs:
+# ``torch.nn.Flatten(start_dim: int = 1, end_dim: int = -1)``
+#
+# Here is an example using one of the training_data set items:
+tensor = training_data[0][0]
+print(tensor.size())
+
+model = nn.Sequential(
+    nn.Flatten()
+)
+flattened_tensor = model(tensor)
+flattened_tensor.size()
+
+##############################################
+# `nn.Linear <https://pytorch.org/docs/stable/generated/torch.nn.Linear.html>`_ to add a linear layer to the model.
+# -------------------------------
+#
+# Now that we have flattened our tensor dimension we will apply a linear layer 
+# transform that will calculate/learn the weights and the bias.
+#
+# From the docs:
+# 
+# ``torch.nn.Linear(in_features: int, out_features: int, bias: bool = True)``
+#
+
+input = training_data[0][0]
+print(input.size())
+model = nn.Sequential(
+    nn.Flatten(),    
+    nn.Linear(28*28, 512),
+)
+output = model(input)
+output.size()
+
+#################################################
+# Activation Functions
+# -------------------------
+#
+# After the first two linear layer we will call the `nn.ReLU <https://pytorch.org/docs/stable/generated/torch.nn.ReLU.html)>`_ 
+# activation function. Then after the third linear layer we call the `nn.Softmax <https://pytorch.org/docs/stable/generated/torch.nn.Softmax.html>`_ 
+# activation to rescale between 0 and 1 and sum to one.
+#
+
+model = nn.Sequential(
+        nn.Flatten(),
+        nn.Linear(28*28, 512),
+        nn.ReLU(),
+        nn.Linear(512, 512),
+        nn.ReLU(),
+        nn.Linear(512, len(classes)),
+        nn.Softmax(dim=1)
+    ).to(device)
+    
+print(model)
+
+
+###################################################
+# Forward Function
+# --------------------------------
+#
+# In the class implementation of the neural network we define a ``forward`` function.  
+# Then call the ``NeuralNetwork`` class and assign the device. When training the model we will call ``model``
+# and pass the data (x) into the forward function and through each layer of our network.
+#
+#
+def forward(self, x):
```

Comment:
@sethjuarez, let's not use the inline approach in this. Since this is a fundamentals 101 guide, I don't think users will be served well with non-standard approaches.

Reply:
@suraj813 I will remove the inline approach from the tutorial